src/backend/tcop/utility.c

```diff
@@ -3729,6 +3729,10 @@ CreateCommandTag(Node *parsetree)
 			tag = CMDTAG_RETRIEVE;
 			break;
 
+		case T_CreateWarehouseStmt:
```

Comment:
handle T_CreateWarehouseStmt in GetCommandLogLevel/ClassifyUtilityCommandAsReadOnly, also make check in other places.